var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -123,6 +124,16 @@ def configure_args(self):
 
         return args
 
+    @when('+static')
+    def build(self, spec, prefix):
+        params = ['V=1']
+        params += self.build_targets
+        with working_dir(self.build_directory):
+            make(*params)
+            make('clean')
+            params += ['LDFLAGS=-all-static']
```

Comment:
It might be worth saying `conflicts('+static', when="os=fedora")` (this might be the case on other systems as well but I think "officially" recording that in the package.py would be very useful).

Reply:
I gave this another try and it was due to me not having installed glibc-static. With this package, I can also statically link on Fedora.